test/Runner.java

```diff
@@ -49,7 +49,7 @@ public Runner() throws InterruptedException, TimeoutException, IOException {
             throw new IllegalArgumentException("TypeDB distribution file is missing from " + distributionArchive.getAbsolutePath());
         }
 
-        checkAndDeleteExistingDistribution(distributionArchive);
+        checkAndDeleteExistingDistribution();
```

Comment:
Do a lightweight refactor of this class by:
1. Placing the methods which are related close to one another.
2. Removing unneeded method arguments.